doc/data.md

```diff
@@ -163,10 +165,53 @@ SDMX-CSV offers the possibility to set the value for two parameters via the medi
         </message:Structure>
     </message:Header>
-    <message:DataSet action="Replace" validFromDate="2014-12-03T15:30:00.000+01:00" structureRef="ECB_RTD1">[...]</message:DataSet>
-    <message:DataSet action="Replace" validFromDate="2015-06-02T15:30:00.000+02:00" structureRef="ECB_RTD1">[...]</message:DataSet>
-    <message:DataSet action="Replace" validFromDate="2015-10-21T15:30:00.000+02:00" structureRef="ECB_RTD1">[...]</message:DataSet>
-    <message:DataSet action="Replace" validFromDate="2016-06-01T15:30:00.000+02:00" structureRef="ECB_RTD1">[...]</message:DataSet>
-    <message:DataSet action="Delete" validToDate="2014-11-05T15:30:00.000+01:00" structureRef="ECB_RTD1">[...]</message:DataSet>
-</message:GenericData>
-```
+    <message:DataSet action="Replace" validFromDate="2014-12-03T15:30:00.000+01:00" structureRef="ECB_RTD1"> 
+    [...]</message:DataSet>
+    <message:DataSet action="Replace" validFromDate="2015-06-02T15:30:00.000+02:00" structureRef="ECB_RTD1"> 
+    [...]</message:DataSet>
+    <message:DataSet action="Replace" validFromDate="2015-10-21T15:30:00.000+02:00" structureRef="ECB_RTD1"> 
+    [...]</message:DataSet>
+    <message:DataSet action="Replace" validFromDate="2016-06-01T15:30:00.000+02:00" structureRef="ECB_RTD1"> 
+    [...]</message:DataSet>
+    <message:DataSet action="Delete" validToDate="2014-11-05T15:30:00.000+01:00" structureRef="ECB_RTD1"> 
+    [...]</message:DataSet>
+  </message:GenericData>
+  ```
+
+- Retrieve attributes, but no data, using the `attributes` and `measures` parameters:
+
+        https://ws-entry-point/data/dataflow/ECB/EXR/1.0.0/M.USD.*.*.*?attributes=all&measures=none
+
+  Services must return **all attributes** relevant for the matching data. For example, let's assume the following DSD (different from previous examples):
+
+  ```
+  Dimensions:
+    FREQ: Frequency
+    CUR1: Currency 1
+    CUR2: Currency 2
+    TIME_PERIOD: Time period
+  Measures:
+    OBS_VALUE: Observation Value
+  Dataflow attributes:
+    UNIT_MULT: Unit multiplier
+  Group attributes:
+    DECIMALS: Number of decimals
+    UNIT_MEAS: Unit of measure
+  Time series attributes:
+    COLL: Collection
+  Observation attributes:
+    OBS_COM: Observation comment
+    OBS_STATUS: Observation status
+  ```
+
+  And let's assume the following dataset:
+  
+  ```
+  FREQ,CUR1,CUR2,TIME_PERIOD,OBS_VALUE,UNIT_MULT,DECIMALS,UNIT_MEAS,COLL,OBS_COM,OBS_STATUS
+  D,CHF,EUR,2021-10-05,1.0752,0,4,CHF,E,,A
+  M,CHF,EUR,2021-09,1.0857,0,4,CHF,A,,A
+  M,USD,EUR,2021-09,1.032,0,4,USD,A,,A
+  ```
+
+  When querying for `D.CHF.*.*`, the first data row would be matched. All attributes applying to this row must be returned, regardless of their attachment level, i.e. the response must include the dataflow-level (`UNIT_MULT`), the group-level (`DECIMALS`, `UNIT_MEAS`), the series-level (`COLL`), and the observation-level (`OBS_STATUS`) attributes. `OBS_COM` can be ignored, as no value is available for this optional attribute.
```

Comment:
Does the key contain the time dimension?

- If yes: then the definition of the key parameter needs to be updated, at least to clarify this in the given example
- If no: then the example in the new explanation for 'Retrieve attributes, but no data' needs to be correct, as `D.CHF.*.*` would have one `*` too much.